src/apify_client/_utils.py

```diff
@@ -201,3 +201,26 @@ def _encode_key_value_store_record_value(value: Any, content_type: Optional[str]
         value = json.dumps(value, ensure_ascii=False, indent=2).encode("utf-8")
 
     return (value, content_type)
+
+
+class ListPage:
+    """A single page of items returned from a list() method."""
+
+    #: list: List of returned objects on this page
+    items: List
+    #: int: Count of the returned objects on this page
+    count: int
+    #: int: The limit on the number of returned objects offset specified in the API call
+    offset: int
+    #: int: The offset of the first object specified in the API call
+    limit: int
+    #: int: Total number of objects matching the API call criteria
+    total: int
+
+    def __init__(self, data: Dict) -> None:
+        """Initialize a ListPage instance from the API response data."""
+        self.items = data['items'] if 'items' in data else []
+        self.offset = data['offset'] if 'offset' in data else 0
+        self.limit = data['limit'] if 'limit' in data else 0
+        self.count = data['count'] if 'count' in data else len(self.items)
+        self.total = data['total'] if 'total' in data else self.offset + self.count
```

Comment:
I think it'd be more canonical to have proper `__init__` accepting individual arguments and use `@classmethod` to create `ListPage` from dict.

```
@classmethod
def from_dict(cls, data):
    return cls(...individual values from data...)
    
To simplify declaring members and then explicitly writing init, you can also use `attrs` package (https://www.attrs.org/en/stable/) which helps you to define this with minimum boilerplate.